scipy/stats/_continuous_distns.py

```diff
@@ -7663,6 +7664,28 @@ def _fitstart(self, data):
 skewcauchy = skewcauchy_gen(name='skewcauchy')
 
 
+# For odd order, the noncentral moments for the skew-normal distribution
+# with location 0 and scale 1 can be expressed as a polynomial in delta,
+# where delta = a/sqrt(1 + a**2) and `a` is the skew-normal shape parameter.
+# This dictionary holds coefficents for that polynomial for orders up to 19.
+_skewnorm_odd_moments = {
```

Comment:
This will slow down the import of `_continuous_distns.py` itself, and thus presumably `scipy.stats`. Instead this could be put in a `@lru_cache`-decorated `_get_skewnorm_odd_moments()` function that returns these values. Then there is no cost at import time, only at the first skewnorm call.

Reply:
The joys of micro-optimizing Python...  I'll give it a shot.

It would be interesting to see how much the import time of any specific module could be improved if we went over the entire code base with an eye towards this type of optimization.

Reply:
Maybe it's not worth it if it's in the microsecond range. But in the past at least I've nested `scipy` submodule imports in functions because they were slow. Hence the idea to at least work toward not making the problem worse...

Reply:
Done.